python/experimental/cudaq/__init__.py

```diff
@@ -102,15 +101,13 @@ def disable_jit():
             print("Could not find a suitable cuQuantum Python package.")
         pass
 
-
-
 initKwargs = {'target': 'qpp-cpu'}
```

Comment:
This logic was removed from here in PR#877, needs to be aligned with mainline.
I see a comment in `_query_gpu.py` - same applies here.